sky/cli.py

```diff
@@ -2673,6 +2673,25 @@ def _hint_or_raise_for_down_spot_controller(controller_name: str):
                        'to terminate (see caveats above).')
 
 
+def _hint_or_raise_for_down_sky_serve_controller(controller_name: str):
+    services = global_user_state.get_services_from_controller_name(
+        controller_name)
+    if services:
+        service_names = [service['name'] for service in services]
+        with ux_utils.print_exception_no_traceback():
+            plural = '' if len(service_names) == 1 else 's'
+            raise exceptions.NotSupportedError(
+                f'{colorama.Fore.RED}Tearing down the sky serve controller '
+                f'is not supported, as it is currently serving the following '
+                f'service{plural}: {", ".join(service_names)}. Please teardown '
+                f'the service{plural} first with {colorama.Style.BRIGHT}sky '
+                f'serve down {" ".join(service_names)}'
+                f'{colorama.Style.RESET_ALL}.')
+    msg = (f'Tearing down sky serve controller: {controller_name}. It will '
+           'increase cold start time for your next `sky serve up` commands.')
```

Comment:
```suggestion
    msg = (f'Tearing down sky serve controller: {controller_name}.')
```
I think it's okay to not show cold start msg.